include/revng/ABI/FunctionType/Layout.h

```diff
@@ -181,4 +182,27 @@ struct Layout {
   void dump() const debug_function;
 };
 
+inline std::span<const model::Register::Values>
+calleeSavedRegisters(const model::CABIFunctionType &Function) {
+  return abi::Definition::get(Function.ABI()).CalleeSavedRegisters();
+}
+inline std::span<const model::Register::Values>
```

Comment:
Leave a blank line between each function. If you really want to group them, use doxygen facilities for grouping.

Reply:
Done.